test/units/testsuite-17.13.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -ex
+set -o pipefail
+
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+# Test for `udevadm control -p`
+
+test_not_property() {
+    [[ -z $(udevadm info --query property --property "$2" --value "$1") ]]
+}
+
+test_property() {
+    [[ $(udevadm info --query property --property "$2" --value "$1") = "$3" ]]
+}
+
+# shellcheck disable=SC2317
+cleanup() {
+    set +e
+
+    udevadm control -p FOO= -p BAR=
+
+    losetup -d "$dev"
+    rm -f "$blk" "$rules"
+}
+
+# Set up a test device
+trap cleanup EXIT
+
+blk="$(mktemp)"
+truncate -s 1M "$blk"
+dev="$(losetup --show -f "$blk")"
```

Comment:
Not a matter, but using /dev/null or so is simpler. See units/testsuite-17.04.sh